parlai/agents/rag/rag.py

```diff
@@ -381,13 +412,16 @@ def update_state_dict(
             init_weight = (
                 model.seq2seq_encoder.position_embeddings.weight  # type: ignore
             )
-            state_dict[key] = torch.cat(
-                [
-                    state_dict[key].to(init_weight),  # type: ignore
-                    init_weight[-opt['n_extra_positions'] :, :],  # type: ignore
-                ],
-                dim=0,
-            )
+            if init_weight.size(0) <= opt['n_positions'] + opt['n_extra_positions']:
```

Comment:
nit: should it be `<` or `<=`. I am guessing for the equal condition we do not need to resize.

Reply:
good catch